src/tests/ftest/util/ec_utils.py

```diff
@@ -406,63 +407,43 @@ def start_online_single_operation(self, operation, parity=1):
 
         # Verify the queue and make sure no FAIL for any run
         while not self.out_queue.empty():
-            if self.out_queue.get() == "FAIL":
+            if (self.out_queue.get() == "FAIL" or self.out_queue.get() == "ERR"):
```

Comment:
online_rebuild_mdtest.py does not use this code path

Reply:
Ok. Will remove the ERR check.